composer.json

```diff
@@ -24,8 +24,8 @@
     "require-dev": {
         "phpstan/phpstan": "^0.11",
         "phpstan/phpstan-phpunit": "^0.11",
-        "phpunit/phpunit": "^7.0",
-        "doctrine/coding-standard": "^1.0",
+        "phpunit/phpunit": "^7.0|^8.5|^9.4",
+        "doctrine/coding-standard": "^6.0|^8.0",
```

Comment:
i will make a seperate PR that updates coding standard first and autofixes everything.